### DISCUSSION TITLE
Use cases wiki and blogpost "First use cases in our wiki"

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#17] [General]
Hey,
comment your feedback about our "First use cases in our wiki" page and the use case documentation mentioned there from your peer-review here:

**Comment by Dominik-Seiler** [comment]
Hi FileFighter-Team,

we find your idea interesting to offer an alternative to Dropbox and Nextcloud, because we often use this service ourselves.
We like your mockups, they are already very well elaborated and detailed.  Also the clean look of your diagrams (and blog) are very clear and easy to understand, keep it that way!

By the Use-Case-Specification "View Content Folder" you could add a search bar, with which the user can quickly find a folder/file, if he has many folders/files. We think this would improve the usability.
We hope you are still expanding your Use-Case-Specifications, because they are not yet complete.

We’re looking forward to the NAS-Application and can’t wait to use it!

Greetings
SimpleSurvey-Team

**Comment by qvalentin** [comment]
@Dominik-Seiler 
Dear SimpleSurvey-Team,
thanks for your feedback. A search function is already planned, we just decided that it should be a separate use case because of technical reasons.

Best regards
FileFighter-Team

**Comment by DSinMeliodas** [comment]
Dear FileFighter-Team,
Your desciption for the Use-Case Specification "Share files and directories (Permissions CRUD)" is quiet good and ur Flow of Events is nice so look at easily to read and is is clear at any point what will happen next. Just as your other Use-Case Specification the second one is also quiet good evne though it is a little bit short you get all the neccassary informations from it which in my opinion is even better.

The only thing I do not understand is why if the User will be redirected towards an error page when having not enough permissions and gets not just shown an error popup, but this is only about the implementation and not about the Use-Case specification.

I think also it is ok to write tbd at points of your Use-Case that aren't yet decided or able to be displayed but don't forgett to update them and do this by centralising everthing because it is easier ;-)

Even though it's late
with best regards
the Futurum Team

**Comment by qvalentin** [comment]
Dear Futurum Team
thank you for your feedback. The point about the error page is valid. Maybe the user will just be redirected to his start page and gets show a popup? But keep in mind that the beginning of the activity was clicking on a URL, where 404 pages are normal if you don't have permissions.

Best regards
FileFighter-Team
